server/memphis_handlers_stations.go

```diff
@@ -310,6 +310,7 @@ func (s *Server) createStationDirectIntern(c *client,
 	err = createStreamFunc()
 	if err != nil {
 		serv.Errorf("createStationDirect: Station " + csr.StationName + ": " + err.Error())
+		fmt.Println("Error: duplicates window can not be larger then max age", error.Error())
```

Comment:
Actually it is not the fix we need, I'll explain myself.
Please notice that we are getting as a parameters retention type + retention value + idempotency window
we have to check whether the retention value is not smaller than the idempotency window in case of retention type from type "message_age_sec".
This validation has to sit in a specific function, something like validateIdempotencyWindow.
You can take a reference from validateReplicas function, a few lines above.

Reply:
and the error handling you can take a reference as well from the way we are handling an error comes back from the validateReplicas function